### ISSUE TITLE
publishPod from CLI: Text isn't output to terminal/console for dendron.markdown or dendron.html, and --vault arg is ignored

### ISSUE THREAD

**Issue by ScriptAutomate** [dendronhq/dendron#2345]
## Please select the area your bug applies to. (Multiple selections are Possible. You can leave blank if you're not sure.)

- [ ] Workspace.  VSCode workspace, vaults, Intellisense/autocomplete, Dendron settings
- [ ] Lookup.  Dendron's Lookup Command
- [ ] Views.  Dendron Preview, Tree View, Side Panels in the UI
- [ ] Schema.  Dendron Schemas
- [x] Pod.  Data import from / export to Dendron
- [ ] Publish.  External Site Publish
- [ ] Markdown.  Markdown features, such as syntax support and features

## Describe the bug

Using the CLI, I'm not able to get `publishPod` to work as expected. This seems to be the case for both Markdown and HTML, the two I have tested.

In the future, we should look at the following to work via CLI:
```sh
dendron publishPod --wsRoot . --podId dendron.html --query root
```
At the moment, it doesn't work, even if the pod was first generated via CLI:
```sh
dendron publishPod --wsRoot . --podId dendron.html --genConfig
```
It also seems to require `vaultName` to be present in the config yml, otherwise it will fail. `--vault` looks to be ignored at the CLI, with or without `vaultName` present in the yml config.

## To Reproduce

1. Run `dendron publishPod --wsRoot . --podId dendron.html --genConfig` to generate the HTML publish pod config
2. Run `dendron publishPod --wsRoot . --vault vault --podId dendron.html --query root` (where `vault` is named `vault`)

This errors out. Looks like `vaultName` is required to be configured in the `dendron.html/config.publish.yml`. After that is configured, I can run the command, but with zero output.

3. Run `dendron publishPod --wsRoot . --podId dendron.html --query root`
4. See zero output. No error and no text output to terminal.

## Expected behavior

Two things were expected (which means this may require two issues?):
- `dendron publishPod --wsRoot . --vault vault --podId dendron.html --query root` should work, even if `vaultName` is not configured in the `.yml`, since I am providing `vault` via `--vault` (which says in the `--help` is a valid arg).
- `dendron publishPod --wsRoot . --vault vault --podId dendron.html --query root` should output the result in the console/terminal output

For another example, I imagine that the following should be the same commands:

```sh
# Output `root.md` as GitHub Flavored Markdown
# This command currently works without issue
dendron note lookup --wsRoot . --query root --vault vault --output md_gfm
```

```sh
# Create config for publishPod
dendron publishPod --wsRoot . --podId dendron.markdown --genConfig
# Output `root.md` as GitHub Flavored Markdown
dendron publishPod --wsRoot . --vault vault --podId dendron.markdown --query root
```

## Desktop (please complete the following information)

- OS: Pop!_OS Linux 21.10
- Dendron CLI Version: `0.80.0`

**Comment by Harshita-mindfire** [comment]
Thanks for reporting the bug!  Will be adding a fix for this in the current sprint.

**Comment by Harshita-mindfire** [comment]
Hi, this has been fixed with Dendron v0.82. Please let us know if it continues to happen, otherwise, this issue will close in a few days.

**Comment by ScriptAutomate** [comment]
> Hi, this has been fixed with Dendron v0.82. Please let us know if it continues to happen, otherwise, this issue will close in a few days.

Commands work!

Follow-up questions, in case I should open new issues:
- Will the `dendron exportPod` commands be able to work for v2, or will there be a new arg like `exportPodv2`, or is there a way for CLI to reference pod v2 configs? I'd like to be able to use the V2 configurations with CLI.
- I couldn't get certain features like `wikilinkToURL` config values to be acknowledged, which I think references `siteUrl` in `dendron.yml`. This works fine with Dendron Workspace commands for PublishPod, and the wikilink is converted in the clipboard as expected, just not via CLI.
  - I tried a workaround with `npx dendron publishPod --wsRoot . --podId dendron.markdown --vault vault --query lang.python --config "wikiLinkToURL"="true"`, but this fails with:

```
  payload: 'error: [{"instancePath":"/wikiLinkToURL","schemaPath":"#/properties/wikiLinkToURL/type","keyword":"type","params":{"type":"boolean"},"message":"must be boolean"}]'
```

**Comment by Harshita-mindfire** [comment]
Hey @ScriptAutomate there will be separate command for podsV2 cli, I'm currently working on it. See [Draft Docs](https://github.com/dendronhq/dendron-site/pull/400/files)
